test/test_typecheck.js

```diff
@@ -15,6 +15,7 @@ function typecheckTest() {
     var code = fs.readFileSync('./test/sample.apps').toString('utf8').split('====');
 
     Q.all(code.map((code) => {
+        if (code.indexOf(`// Meta language`) < 0) return Q();
```

Comment:
Left-over debugging code (and probably what's causing the coverage drop)

Reply:
removed